sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessorClientBuilder.java

```diff
@@ -916,4 +921,39 @@ public void close(CloseContext closeContext) {
         };
     }
 
+    private static EventHubClientBuilder copyOptions(EventHubClientBuilder source) {
+        final EventHubClientBuilder builder = new EventHubClientBuilder()
+            .clientOptions(source.getClientOptions())
+            .configuration(source.getConfiguration());
```

Comment:
nit; wondering if the readability improves if we combine all setters for those we don’t have to (null/empty) check the `source` value

```java   
     final EventHubClientBuilder builder = new EventHubClientBuilder()
            .clientOptions(source.getClientOptions())
            .configuration(source.getConfiguration())
            .proxyOptions(source.getProxyOptions())
            .transportType(source.getTransportType())
            .retryOptions(source.getRetryOptions())
            .consumerGroup(source.getConsumerGroup())
            .scheduler(source.getScheduler())
            .verifyMode(source.getVerifyMode());
```